sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/CSharpExplodedGraph.cs

```diff
@@ -358,13 +358,18 @@ protected override void VisitInstruction(ExplodedGraphNode node)
                     {
                         var stringLiteralExpression = instruction as LiteralExpressionSyntax;
                         var sv = new SymbolicValue();
-                        if (stringLiteralExpression.Token.ValueText == "")
+                        var valueText = stringLiteralExpression.Token.ValueText;
+                        if (string.IsNullOrEmpty(valueText))

```

Comment:
from live feedback:
Q:why check for `null` as well?
A: because it's a literal expression and the string cannot be null inside it